packages/typespec-go/src/tcgcadapter/clients.ts

```diff
@@ -34,7 +34,7 @@ export class clientAdapter {
     }
     for (const sdkClient of sdkPackage.clients) {
       // start with instantiable clients and recursively work down
-      if (sdkClient.initialization.access === 'public') {
+      if (sdkClient.clientInitialization.initializedBy & tcgc.InitializedByFlags.Individually) {
```

Comment:
I see these are flags now which usually implies that one or both can be set. Is that true?